MidasCivil_Engine/Compute/CombineTextFiles.cs

```diff
@@ -117,17 +124,35 @@ public static bool CombineTextFiles(string filePath, List<Type> types = null, bo
                 }
 
                 independents.Insert(0, "REBAR-MATL-CODE");
-                independents.Insert(0, "UNIT");
-                independents.Insert(0, "VERSION");
                 independents.Add("LOAD-GROUP");
                 independents.Add("LOADCOMB");
 
                 List<string> loadcases = Directory.GetDirectories(directory).ToList();
+                loadcases.Remove(directory + "\\00_MetaData");
 
                 using (var combined = File.Create(path))
                 {
                     using (StreamWriter writer = new StreamWriter(combined))
                     {
+                        writer.Write(intro);
+                        writer.Flush();
+                        foreach (string file in MetaData)
+                        {
+                            if (File.Exists(directory + "\\00_MetaData\\" + file + ".txt"))
+                            {
+                                using (var input = File.OpenRead(directory + "\\00_MetaData\\" + file + ".txt"))
+                                {
+                                    if (new FileInfo(directory + "\\00_MetaData\\" + file + ".txt").Length != 0)
+                                    {
+                                        input.CopyTo(combined);
+                                        input.Close();
+                                    }
+                                }
+                                writer.Write(System.Environment.NewLine);
+                                writer.Flush();
+                            }
+                        }
```

Comment:
All the .txt files should sit in \TextFiles\. The only exceptions being Loadcases and the associated loads.

Reply:
![image](https://user-images.githubusercontent.com/55385899/118116216-ea02d600-b3e1-11eb-87fd-866623019be0.png)
As above